examples/temperature-sensor/temperature-publisher/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "dilan"
+name = "temperature_publisher"
+version = "0.1.0"
+distribution = "2201.7.1"
```

Comment:
Do we need to add a distribution version here ?

Reply:
Fixed